src/main/java/gov/nih/nci/bento_ri/service/InventoryESService.java

```diff
@@ -189,6 +190,20 @@ public Map<String, Object> buildFacetFilterQuery(Map<String, Object> params, Set
                         treatment_response_filters.add(Map.of(
                             "terms", Map.of("treatment_responses." + key, valueSet)
                         ));
+                    } else if (PARTICIPANT_PARAMS.contains(key) && !indexType.equals("participants")) {// Filter by nested Participant property
+                        filter.add(Map.of(
+                            "nested", Map.of(
+                                "path", "participant",
+                                "query", Map.of(
+                                    "bool", Map.of(
+                                        "filter", List.of(Map.of(
+                                            "terms", Map.of("participant." + key, valueSet)
+                                        ))
+                                    )
+                                ),
+                                "inner_hits", Map.of()
+                            )
+                        ));
```

Comment:
This is the most noteworthy change in the file. We must construct a "nested" clause in the Opensearch query for any Participant property that would be found as a nested property in the documents of an index.